contributing/documentation/overview.rst

```diff
@@ -258,6 +258,29 @@ link displayed for Platform.sh service.
     Only Pull Requests to maintained branches are automatically built by
     Platform.sh. Check the `roadmap`_ for maintained branches.
 
+Build the Documentation Locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Alternatively you can build the documentation in your own computer following
+these steps:
```

Comment:
To be honest, I was confused in thinking that could build the documentation locally and use it completely "offline", which it's not possible right now and probably never will https://github.com/symfony/symfony-docs/issues/6844#issuecomment-237632673. Maybe we can add the subject "only for testing purpose" somehow? thanks